### DISCUSSION TITLE
Hacktoberfest 2021

### DISCUSSION THREAD

**Discussion by megamayoy** [pytest-dev/pytest#9188] [General] [answered]
Hey, everyone
I wanna contribute to this project during the Hacktoberfest. It's my first time to contribute here. I would appreciate it if anyone suggests issues to work on for new contributors or put the hacktoberfest label on issues that can be done within this month.
Thanks!

**Comment by nicoddemus** [comment, marked as answer]
Hi @megamayoy,

We have an "easy" label marking issues that should be simple for new contributors, I suggest to start there. 👍
